client/state/selectors/is-jetpack-golden-token-pending-activation.ts

```diff
@@ -0,0 +1,3 @@
+export const isJetpackGoldenTokenPendingActivation = () => {
+	return true;
```

Comment:
Worth adding a TODO comment here, so we don't accidentally leave this unfinished.